### PR TITLE
Fill out usage and some general documentation

### PR DESCRIPTION
**Pull request by Samwalton9**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,25 @@
-# WorklistTool-GSoC-2018
-Tool is being developed for Wikimedia under GSoC 2018
+# Worklist Tool
+The Worklist Tool is a tool for creating lists of Wikimedia pages which users can then assign themselves to for claiming and tracking work during editing events and programs.
+
+The tool was first developed with Wikimedia during Google Summer of Code 2018.
+
+## Usage
+The Worklist Tool is currently hosting on Toolforge at https://tools.wmflabs.org/worklist-tool/.
```
Comment (eggpi) [inline README.md]: Just "The Worklist Tool is hosted on Toolforge" ... ?

(Also we should consider a catchier name, but don't have to do it now).

**Comment by eggpi** [comment]
Nice, thank you! I've just left one minor comment. @MeghaSharma21 what do you think?

**Comment by MeghaSharma21** [comment]
Thanks for the effort :). Overall it looks good but I have some comments - 
1. I feel we should expand more on how to create a worklist by including what is expected in description field, tags field etc.
2. We can include some information about Petscan and why it has been integrated with the tool.

File: README.md (hunk 1 above)
Comment (Samwalton9) [reply README.md]: Yep, that was some poor wording - fixed.

Comment (Samwalton9) [reply README.md]: Yep, that was some poor wording - fixed.

**Comment by Samwalton9** [comment]
Great - just added a couple of commits that should address that feedback :)

**Comment by MeghaSharma21** [comment]
It's nice! Feel free to merge it.

**Pull request merged**